openedx/core/djangoapps/programs/utils.py

```diff
@@ -17,8 +17,7 @@
 from openedx.core.djangoapps.catalog.utils import (
     get_programs as get_catalog_programs,
     munge_catalog_program,
-    get_run_marketing_url,
-)
```

Comment:
Revert this change. The trailing comma is meant to leave the next person with a clean diff if they need to add another import.